src/components/delegates/delegates.less

```diff
@@ -36,6 +36,9 @@ delegates {
   .pending {
     background-color: #eaeae9;
   }
+  .voted{
```

Comment:
`.voted` should be defined before `.pending` in order to have the pending vote gray, not blue.